Resources/public/js/views/universaldiscovery/ez-universaldiscoverysearchview.js

```diff
@@ -0,0 +1,500 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-universaldiscoverysearchview', function (Y) {
+    "use strict";
+    /**
+     * Provides the universal discovery search method
+     *
+     * @module ez-universaldiscoverysearchview
+     */
+    Y.namespace('eZ');
+
+    var events = {
+            '.ez-ud-search-button': {
+                'tap': '_search',
```

Comment:
it would be very nice to also trigger a search if the user types "Return" on his keyboard (I did it several times by habits when I tested the PR :))